Gemfile

```diff
@@ -60,8 +60,9 @@ group :development, :test do
   gem "rspec-rails", "~> 3.5"
   gem "rspec-request_describer"
   gem "rubocop"
+  gem "codecov", require: false
```

Comment:
__[RuboCop]__ Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `codecov` should appear before `rubocop`. ([link](https://sider.review/gh/repos/105680869/pulls/99#issue-3973714))

<sub>posted by Sider</sub>